src/main.rs

```diff
@@ -316,7 +309,18 @@ impl<'args> Exa<'args> {
             file_style: file_name::Options { quote_style, .. },
             ..
         } = self.options.view;
-        for dir in dir_files {
+        for mut dir in dir_files {
+            let dir = match dir.read() {
+                Ok(dir) => dir,
+                Err(e) => {
+                    if e.kind() == ErrorKind::PermissionDenied {
+                        exit(exits::PERMISSION_DENIED);
+                    };
```

Comment:
Correct me if I’m wrong, but that means a single “permission denied” will terminate eza, right? It’s modifying eza’s behavior in a way that I think is not desirable.

Reply:
Yes, you're correct. Do you suggest we skip such dirs?

Reply:
Hum I just checked and `eza /*` just prints `"/lost+found": Permission denied (os error 13)`. So maybe try that or something that triggers the same error, and check that the behavior is still the same?

Reply:
Looking back, I think even the original behavior with exiting isn't a great idea. So In my latest commit,`eza` should continue even if an unreadable folder is encountered and logs detailed info about those folders to the user/caller.

https://github.com/eza-community/eza/pull/1419/commits/a622db355b8cf447aa59bd25cb1d449ca40c08f6

Reply:
I think skipping the paths but listing them at the end is a nice idea, we’ll see if others people think the same.

I personally think it would be better with a `:` after “denied” and the paths listed without `-` (but keeping a two spaces indentation).

Also, please edit your commit messages to conform to [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) (e.g. by using `git rebase --interactive main`), so that the tests can run in the CI and check it doesn’t break anything! (don’t hesitate to ask if you need help).

Reply:
![Screenshot from 2025-05-03 18-37-53](https://github.com/user-attachments/assets/aad67bbc-0814-4b1a-8792-b7b543429a69)